pkg/k8s/k8s_role.go

```diff
@@ -29,3 +30,29 @@ func (c *Client) CreateRole(
 func (c *Client) DeleteRole(ctx context.Context, name string) error {
 	return c.clientset.RbacV1().Roles(c.namespace).Delete(ctx, name, metav1.DeleteOptions{})
 }
+
+func (c *Client) CreateClusterRole(
+	ctx context.Context,
+	name string,
+	labels map[string]string,
+	policyRules []rbacv1.PolicyRule,
+) error {
+	_, err := c.clientset.RbacV1().ClusterRoles().Get(ctx, name, metav1.GetOptions{})
+	if err == nil || !errors.IsNotFound(err) {
+		return ErrClusterRoleAlreadyExists.WithParams(name)
```

Comment:
We should include the error since the cases where it is not nil we want to understand the error. 
```suggestion
		return ErrClusterRoleAlreadyExists.WithParams(name, err)
```